.github/scripts/install-tar-wrapper.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright (c) 2019 Kevin R Croft <krcroft@gmail.com>
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# This script creates a simple wrapper for tar that performs as follows
+# when run under a GitHub workflow:
+#   * always returns success
+#   * replaces gzip with zstd
```

Comment:
Might work! GitHub's cache action looks for 'tar' in the path, so if we can override it's path (if env: is a valid key in with: items), then this would work. Will try!

Reply:
Tentative fix in 52bceb0c